configs/docusaurus_demo.json

```diff
@@ -59,4 +59,4 @@
     "415526327"
   ],
   "nb_hits": 1014
```

Comment:
This too? Does this need to be changed to for docusaurus_demo?

Reply:
It will be changed automatically based on how many hits this PR will introduce